File: branch_forge/server/Models/Story.cs
namespace branch_forge.Models;

public class Story : RepoItem<int>
{
  public string Description { get; set; }
  public string Title { get; set; }
  public string CoverImg { get; set; }
  public int LikeCount { get; set; }
  public string AuthorId { get; set; }
}

File: branch_forge/client/src/models/Story.js
import { Profile } from "./Account.js"

export class Story {
  constructor(data) {
    this.id = data.id
    this.createdAt = new Date(data.createdAt)
    this.updatedAt = new Date(data.updatedAt)
    this.title = data.title
    this.description = data.description
    this.coverImg = data.coverImg
    this.authorId = data.authorId
    this.likeCount = data.likeCount
    this.creator = new Profile(data.creator)
  }
}